scapy/contrib/tacacs.uts

```diff
@@ -0,0 +1,190 @@
++ Tacacs+ header
+
+= default instanciation
+
+pkt = IP()/TCP(dport=49)/TacacsHeader()
+str(pkt) == b'E\x00\x004\x00\x01\x00\x00@\x06|\xc1\x7f\x00\x00\x01\x7f\x00\x00\x01\x001\x001\x00\x00\x00\x00\x00\x00\x00\x00P\x02 \x00\xd0p\x00\x00\xc0\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00'
+
+= default values build
+
+pkt = IP()/TCP(dport=49)/TacacsHeader()
+pkt.session_id == 0
```

Comment:
Could you rewrite it as follows ? And apply the same pattern to all tests.
`TacacsHeader in pkt pkt.session_id == 0`